### ISSUE TITLE
Improve fitness function to be more accurate when calculating it

### ISSUE THREAD

**Issue by wmartins** [famw/SpelunkBots#15]
We need to improve our fitness function. As discussed, we can try testing two different fitness functions:
- D/T (Distance / Time)
- [Harmonic mean](https://en.wikipedia.org/wiki/Harmonic_mean) between distance and time
- Manhattan distance (`Md = |x2 - x1| + (|y2 - y1|)*2` or `Md = |x2 - x1| + (|y2 - y1|)^2`)

**Comment by martinduartemore** [comment]
As discussed, we could also improve our distance fitness by giving more weight to the **y axis** on the [manhattan distance](https://en.wikipedia.org/wiki/Taxicab_geometry) calculation. Something like this, for example:

```
Md = |x2 - x1| + (|y2 - y1|)*2
```
or this:
```
Md = |x2 - x1| + (|y2 - y1|)^2
```

**Comment by wmartins** [comment]
Closed by @martinduartemore on 6f6e225377c0f6abcd0a72f30447e030a691382f.
